cores/esp8266/Esp.h

```diff
@@ -210,7 +210,20 @@ class EspClass {
         static String getResetInfo();
         static struct rst_info * getResetInfoPtr();
 
-        static bool eraseConfig();
+        /*
+          Erases 4 sectors at the end of flash, 1 - RF_CAL and 3 - SYSTEMPARM.
+          These are the same additional sectors that are erase when you select
+          Erase Flash: "Sketch + WiFi Settings" from the Arduino IDE Tools menu.
+
+          As a precaution, since this operation erases the running SDKs flash
+          configuration space, use reset flag "true" with eraseConfig. Also, for
+          additional protection, call "WiFi.mode(WIFI_OFF)" before calling.
+
+          If you need to erase "WiFi Settings" and reboot consider using
+          "ArduinoOTA.eraseConfigAndReset()" it handles shutting down WiFi
+          before the erase.
+        */
+        static bool eraseConfig(bool reset = false);
```

Comment:
Same sentiment as in ArduinoOTA, two separate funcs?
`eraseConfig()` and `eraseConfigAndReset()`

Reply:
done